### PR TITLE
imx7ulp initial support on 3.15

### PR DESCRIPTION
**Pull request by vanmaegima**

### CODE REVIEW

**Comment by vanmaegima** [comment]
This is missing the PM support.

File: core/arch/arm/plat-imx/conf.mk [hunk 1]
```diff
@@ -148,7 +149,8 @@ $(call force,CFG_MX7ULP,y)
 $(call force,CFG_TEE_CORE_NB_CORE,1)
 $(call force,CFG_TZC380,n)
 $(call force,CFG_CSU,n)
-$(call force,CFG_NXP_CAAM,n)
+$(call force,CFG_NXP_CAAM,y)
+$(call force,CFG_IMX_CAAM,n)
```
Comment (Tim-Anderson) [inline core/arch/arm/plat-imx/conf.mk:153]: I am confused here. You have both `y` and `n` set for `CFG_NXP_CAAM`

Comment (vanmaegima) [reply core/arch/arm/plat-imx/conf.mk:153]: CFG_NXP_CAAM is set to **y**, while IMX_CAAM (different driver) is set to **n**
I believe IMX_CAAM is a reduced version of NXP_CAAM

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**